v2/tools/generator/internal/armconversion/convert_from_arm_function_builder.go

```diff
@@ -10,6 +10,8 @@ import (
 	"go/token"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
minor: collapse into the middle "imports from elsewhere" section next to `dave/dst`

Reply:
Done.